README.md

```diff
@@ -1,25 +1,102 @@
-# README
+📗 Table of Contents
```

Comment:
Great job @Ngufor-emmanuella 🤝

- [ ] Kindly restructure your readme file properly so that it can replicate the [readme template](https://github.com/microverseinc/readme-template) provided. You may also customize it according by adding the necessary sections and links.